homeassistant/components/binary_sensor/rflink.py

```diff
@@ -0,0 +1,102 @@
+"""
+Support for Rflink binary sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.rflink/
+"""
+import logging
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASSES_SCHEMA, PLATFORM_SCHEMA, BinarySensorDevice)
+from homeassistant.components.rflink import (
+    CONF_ALIASES, CONF_DEVICES, RflinkDevice, cv, vol)
+from homeassistant.const import (
+    CONF_FORCE_UPDATE, CONF_NAME, CONF_DEVICE_CLASS)
+import homeassistant.helpers.event as evt
+
+CONF_OFF_DELAY = 'off_delay'
+DEFAULT_FORCE_UPDATE = False
+
+DEPENDENCIES = ['rflink']
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_DEVICES, default={}): {
```

Comment:
I don't see `automatic_add` anywhere?

Reply:
automatic_add only works and is only supported for light.rflink and sensor.rflink because rflink has no detection of device type.

Reply:
Ok, then just update the config section in the PR description.

Reply:
Done!